### PR TITLE
feat(p-text-input): add size props to textInput

### LINKED ISSUE #152: [TextInput] Add size prop
**Problem**
Need smaller input.

**Solution**
Add a new property `size`
The old input(height 2rem) is named `md`, new smaller input(height 1.5rem) is named `sm`

**Figma / Jira Link**
[Go to figma](https://www.figma.com/file/IS6P8y1Wn2nfBC4jGlSiya/Components?node-id=10%3A193213&t=0PD8i2VotHsI9PHu-11)

**Additional Description**
Used for custom dashboard's label management flow.
scheduled for v1.10.5 sprint

### PR DESCRIPTION
**Pull request by song2kim**
### To Reviewers
- [ ] Skipping review is not a problem (```style```, ```chore``` ONLY)
- [X] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [X] Feature improvement
  - [ ] These changes require a usage change
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [X] Updated Storybook documents
- [ ] Tested with console(if usages are changed) 

### Description
* add size props to textInput

image

### CODE REVIEW

**Review by yuda110 — APPROVED** [review]
LGTM!!!

File: src/inputs/input/PTextInput.stories.mdx [hunk 1]
```diff
@@ -7,7 +7,7 @@ import { reactive, toRefs } from 'vue';
 import {
     getTextInputArgTypes
 } from "@/inputs/input/story-helper";
-import {getTextInputMenu} from "@/inputs/input/mock";
+import {getTextInputMenu} from "@/inputs/input/mock"; import {INPUT_SIZE} from "@/inputs/input/config";
```
Comment (WANZARGEN) [inline src/inputs/input/PTextInput.stories.mdx]: nitpick: Please add line break 😅

File: src/inputs/input/PTextInput.stories.mdx [hunk 2]
```diff
@@ -398,6 +399,31 @@ export const Template = (args, { argTypes }) => ({
...
 
+## Size
+<Canvas>
+    <Story name="Size" >
+        {{
+            components: { PTextInput },
+            template: `
+                <div  style="display:flex; flex-direction: column; row-gap: 1.5rem;">
+                    <p-text-input v-for="size in sizes" :key="size" :size="size" v-model="value" :placeholder="size" />
+                </div>
```
Comment (WANZARGEN) [inline src/inputs/input/PTextInput.stories.mdx:411]: praise: Nice story 👍

File: src/inputs/input/config.ts [hunk 3]
```diff
@@ -0,0 +1,5 @@
+export const INPUT_SIZE = {
+    sm: 'sm',
+    md: 'md',
+} as const;
+export type INPUT_SIZE = typeof INPUT_SIZE[keyof typeof INPUT_SIZE];
```
Comment (WANZARGEN) [inline src/inputs/input/config.ts]:
```suggestion
export type InputSize = typeof INPUT_SIZE[keyof typeof INPUT_SIZE];
```

File: src/inputs/input/PTextInput.vue [hunk 4]
```diff
@@ -159,6 +161,13 @@ export default defineComponent<TextInputProps>({
             type: [String, Number],
             default: undefined,
         },
+        size: {
+            type: String,
```
Comment (WANZARGEN) [inline src/inputs/input/PTextInput.vue]:
```suggestion
            type: String as PropType<InputSize>,
```

File: src/inputs/input/PTextInput.vue [hunk 5]
```diff
@@ -136,6 +137,7 @@ interface TextInputProps {
     exactMode: boolean;
     useAutoComplete: boolean;
     maskingMode: boolean;
+    size?: string;
```
Comment (WANZARGEN) [inline src/inputs/input/PTextInput.vue]:
```suggestion
    size?: InputSize;
```

**Review by WANZARGEN — APPROVED** [review]
Overall, looks good to me :)
I left some nitpick comments and suggestions~

**Comment by song2kim** [comment]
Thnx a lot your comments! :)

**Pull request merged**